assembl/static2/tests/unit/pages/synthesis.spec.jsx

```diff
@@ -7,6 +7,8 @@ describe('Synthesis component', () => {
   it('should match Synthesis snapshot', () => {
     const props = {
       routeParams: { slug: 'slugId' },
+      // TODO update this to reflect query change. Changing this
```

Comment:
Have you tried again fixing this before merge?

Reply:
@ayyazdaniaryan we can look at it today together if you have time.

Reply:
Actually the test is fine, no need to change anything here, we are testing DumbSynthesis. I removed the comment in https://github.com/assembl/assembl/commit/8e5fb260f2978804f85d65f944e96936b1de4155